go.sum

```diff
@@ -312,6 +316,8 @@ github.com/stretchr/testify v1.8.2 h1:+h33VjcLVPDHtOdpUCuF+7gSuG3yGIftsP1YvFihtJ
 github.com/tonglil/buflogr v1.0.1 h1:WXFZLKxLfqcVSmckwiMCF8jJwjIgmStJmg63YKRF1p0=
 github.com/vertica/vcluster v0.0.0-20230825142712-91bc5a9af516 h1:FGetau7wYqMCFHnfHuW5oM0swH9gQ1aJ1C/4J7V4vso=
 github.com/vertica/vcluster v0.0.0-20230825142712-91bc5a9af516/go.mod h1:7oNKoLThwbNX69xUwZZOdiI3WunMCth3mCF0EXOqtuc=
+github.com/vertica/vcluster v0.0.0-20230901172157-786ef089e17f h1:Y9iiDDHf2RaoE7eVqKIxpPJevY0q2Qj68vTA0JU41kc=
```

Comment:
Can you run `go mod tidy` to remove the old vcluster entries out?

Reply:
Weird, I ran "go mod tidy". Will tidy it again, and rebase on the master.